src/main/java/com/aws/greengrass/deployment/model/Deployment.java

```diff
@@ -83,6 +84,16 @@ public Deployment(DeploymentDocument deploymentDetails, DeploymentType deploymen
         this.deploymentStage = deploymentStage;
     }
 
+    public String getDeploymentUuid() {
+        return Objects.nonNull(deploymentDocumentObj) ? deploymentDocumentObj.getDeploymentId()
+                : null;
+    }
```

Comment:
This is a little confusing, why are we calling it DeploymentUuid? it's the deployment id right?

Reply:
As discussed offline, renamed to `ggDeploymentId` and added comments in the model class